src/ra.c

```diff
@@ -499,7 +499,7 @@ bool ra_process(void)
 				if ((pinfo->nd_opt_pi_flags_reserved & ND_OPT_PI_FLAG_ONLINK) &&
 				    !ptp_link)
 					changed |= odhcp6c_update_entry(STATE_RA_ROUTE, entry,
-									7200, ra_holdoff_interval);
+									0, ra_holdoff_interval);
```

Comment:
This part regards when the onlink bit is set. But your commits mentions RFC 8461 - which regards MTA. Did you mean RFC 4861?????

In any case, the RFC 2462 part you refer to is updated by https://www.rfc-editor.org/rfc/rfc4862#section-5.5.3 (and potentially later RFC)

And that regards the *Autonomous flag* and not the *onlink flag*.

So I'm not sure why we set it to 0 here.